### ISSUE TITLE
import error with 2.1

### ISSUE THREAD

**Issue by Vo-Shay** [blakev/python-syncthing#15]
Installed 2.1 with `pip install syncthing`, but trying `from syncthing import Syncthing` produces the following error:
```
Traceback (most recent call last):
  File "./syncthing", line 3, in <module>
    from syncthing import Syncthing
  File "/usr/local/lib/python2.7/dist-packages/syncthing/__init__.py", line 748
    def __init__(self, *args, last_seen_id=None, filters=None, limit=None, **kwargs):
                                         ^
SyntaxError: invalid syntax
```
Reverting the installation back to 2.0.2 works no problem. 

This is on a Debian 8.8 box, using python 2.7.9

**Comment by blakev** [comment]
Thanks! You're right, I used a Python-3 only feature, I'll make sure it's fixed.

**Comment by blakev** [comment]
@Vo-Shay this is now fixed in 2.1.1 -- thanks again

**Comment by Vo-Shay** [comment]
No problem. Might have some more from me soon ;)
